lib/middleware/websocket/room.rb

```diff
@@ -0,0 +1,16 @@
+module Websocket
+  class Room
+    attr_reader :id
+
+    def initialize(id:)
+      @id = id
+      @clients = []
+    end
+
+    def call; end
```

Comment:
Is this a model, or a service object?

Reply:
@jgayfer It should be a model, it will inevitably have more jobs than a service object would, what are you suggesting?

Reply:
@jgayfer call method has been removed.